src/interactivity/win32/windowUiaProvider.cpp

```diff
@@ -38,8 +30,8 @@ WindowUiaProvider* WindowUiaProvider::Create(IConsoleWindow* baseWindow)
         CONSOLE_INFORMATION& gci = g.getConsoleInformation();
         IUiaData* uiaData = &gci.renderData;
 
-        pScreenInfoProvider = new ScreenInfoUiaProvider(uiaData, pWindowProvider);
-        pWindowProvider->_pScreenInfoProvider = pScreenInfoProvider;
+        pScreenInfoProvider = WRL::Make<ScreenInfoUiaProvider>(uiaData, pWindowProvider);
+        pWindowProvider->_pScreenInfoProvider.Attach(pScreenInfoProvider.Detach());

```

Comment:
This has unclear ownership semantics. This function returns a raw pointer that looks like it's _not owning a reference_. This is based on the use of `Attach` (attaching = "stealing a reference" or "giving ownership to the ComPtr")